sparse_dot_topn/util.py

```diff
@@ -0,0 +1,38 @@
+"""
+Helper functions for name matching
+"""
+import numpy as np
+
+
+def get_n_top_sparse(mat, n_top=10):
```

Comment:
This seems to be only needed for the unit tests. Then it is better to have it under tests folder.

Reply:
Moved.